### DISCUSSION TITLE
@绳网作者 绳网等级的灵感我替你想出来辣！

### DISCUSSION THREAD

**Discussion by fairy-B** [share121/inter-knot#1199] [其他]
按积分制，发贴+10，回复+3，至于每一级经验峰值是多少，用游戏内的峰值/100即可！！（进度条和经验实时显示对您来说应该不难）
可以在每个人的用户名后加上用户的绳网等级（此条可以不采纳）
![image](https://github.com/user-attachments/assets/bc2a6f86-9ea4-4b28-a5d5-71db0b850bea)

**Comment by keerdun** [comment]
+3

**Comment by capacitor1** [comment]
+3

**Comment by ChenggeYouyang** [comment]
+10086

**Comment by lllliufeng** [comment]
会成氵神争霸赛的吧

**Comment by capacitor1** [comment]
> Reply::会成氵神争霸赛的吧

芙芙争霸赛（无端

**Comment by Sunny-Ding110** [comment]
那不是疯狂灌水

**Reply by L502567** [reply]
给没有意义评论出个限制就行

**Comment by suiry-D** [comment]
水+3

**Comment by FIREFLY26710-WEB** [comment]
+3

**Comment by yaotian-yyds** [comment]
+3

**Comment by Xuanan0** [comment]
+3

**Comment by YWphosphenes02** [comment]
你这新点子有点老啊

**Comment by Yitaruose-BaiXu** [comment]
+3

**Comment by cpu644** [comment]
那还得搞个冷却/辨别机制防止水帖

**Comment by 1451374815** [comment]
+3

**Comment by yileinafs** [comment]
+3

**Comment by haifanhen6** [comment]
@share121 
<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >

**Comment by VacuolePaoo** [comment]
# 肥肠抱歉，无法实现
